cmd/agentctl/main.go

```diff
@@ -0,0 +1,66 @@
+// Command agentctl provides utilities for interacting with Grafana Cloud Agent
+package main
+
+import (
+	"os"
+
+	// Adds version information
+	_ "github.com/grafana/agent/pkg/build"
+	"github.com/prometheus/common/version"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/grafana/agent/pkg/agentctl"
+	"github.com/grafana/agent/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	var cmd = &cobra.Command{
+		Use:     "agentctl",
+		Short:   "Tools for interacting with the Grafana Cloud Agent",
+		Version: version.Print("agentctl"),
+	}
+	cmd.SetVersionTemplate("{{ .Version }}\n")
+
+	cmd.AddCommand(
+		configSyncCmd(),
+	)
+
+	_ = cmd.Execute()
+}
+
+func configSyncCmd() *cobra.Command {
+	var (
+		agentAddr string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "config-sync [directory]",
+		Short: "Sync config files from a directory to an Agent's config management API",
```

Comment:
If I understand this correctly, we use the filename as the basis for configuration identifier. I'd make note of that either here or as part of the documentation as it is quite an important detail.